Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openjdk:11-jdk-slim-buster AS build
+
+ARG RSK_RELEASE
+ENV RSK_VERSION $RSK_RELEASE
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y && \
+    apt-get install -y git curl gnupg
+
+RUN useradd -ms /bin/bash rsk
+USER rsk
+
+WORKDIR /home/rsk
+COPY --chown=rsk:rsk . ./
+
+RUN gpg --keyserver https://secchannel.rsk.co/SUPPORT.asc --recv-keys 1DC9157991323D23FD37BAA7A6DBEAC640C5A14B && \
+    gpg --verify --output SHA256SUMS SHA256SUMS.asc && \
+    sha256sum --check SHA256SUMS && \
+    ./configure.sh && \
+    ./gradlew --no-daemon clean build -x test && \
+    cp "build/libs/federate-node-$RSK_VERSION-all.jar" rsk.jar
+
+FROM openjdk:11-jre-slim-buster
+LABEL org.opencontainers.image.authors="ops@iovlabs.org"
+
+RUN useradd -ms /sbin/nologin -d /var/lib/rsk rsk
+USER rsk
+
+WORKDIR /var/lib/rsk
+COPY --from=build --chown=rsk:rsk /home/rsk/rsk.jar ./
+
+ENV DEFAULT_JVM_OPTS="-Xss4M"
+ENV RSKJ_SYS_PROPS="-Dlogback.configurationFile='/etc/rsk/logback.xml' -Drsk.conf.file=/etc/rsk/node.conf"
```

Comment:
Is this the correct path for `logback.xml` and `node.conf` files?

C.C.: @nathanieliov @jeremy-then @marcos-iov

Reply:
Seems correct to me.